pkg/route/api/validation/validation.go

```diff
@@ -85,5 +85,25 @@ func validateTLS(route *routeapi.Route) fielderrors.ValidationErrorList {
 		msg := fmt.Sprintf("invalid value for termination, acceptable values are %s, %s, %s, or emtpy (no tls specified)", routeapi.TLSTerminationEdge, routeapi.TLSTerminationPassthrough, routeapi.TLSTerminationReencrypt)
 		result = append(result, fielderrors.NewFieldInvalid("termination", tls.Termination, msg))
 	}
+	result = append(result, validateNoDoubleEscapes(tls)...)
 	return result
 }
+
+// validateNoDoubleEscapes ensures double escaped newlines are not in the certificates.  Double
+// escaped newlines may be a remnant of old code which used to replace them for the user unnecessarily.
+func validateNoDoubleEscapes(tls *routeapi.TLSConfig) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+	if strings.Contains(tls.CACertificate, "\\n") {
```

Comment:
Yeah, but certs can also have non-PEM content preceding PEM blocks that lists plain-text versions of the certs. If this is a short-term thing to weed out things that cargo-culted our examples, that's fine, but add a TODO to remove the validation at some point... doesn't seem like the sort of thing we'd leave long-term